Classes/Settings.cs

```diff
@@ -490,5 +481,32 @@ protected static void SetValue(string propertyName, string value)
                 }
             }
         }
+
+        private static string GetValuePostfix(Model.Programme progInfo, string name, string defaultValue)
+        {
+            string value = null;
+            if (progInfo != null)
+            {
+                value = GetValue(name + progInfo.Progid.ToString(CultureInfo.InvariantCulture));
```

Comment:
But then a not much used feature gets into the structure of the database. And the settings table is used for other settings that have no business with progid. Who knows what else might pop up in the same way. Those settings might be better added to the programmes table, and the values might be added to progInfo.